pkg/cmd/pulumi/package_gen_sdk.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2016-2022, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	javagen "github.com/pulumi/pulumi-java/pkg/codegen/java"
+
+	"github.com/pulumi/pulumi/pkg/v3/codegen/dotnet"
+	gogen "github.com/pulumi/pulumi/pkg/v3/codegen/go"
+	"github.com/pulumi/pulumi/pkg/v3/codegen/nodejs"
+	"github.com/pulumi/pulumi/pkg/v3/codegen/python"
+	"github.com/pulumi/pulumi/pkg/v3/codegen/schema"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/util/cmdutil"
+)
+
+func newGenSdkCommand() *cobra.Command {
+	var language string
+	var out string
+	cmd := &cobra.Command{
+		Use:   "gen-sdk <schema_source>",
```

Comment:
`gen-sdk` is probably the best verb I do wonder, though, if there are any other names we should consider like just `generate` or `gen`.

Reply:
I considered `generate`, but it wasn't clear what was being generated (sdk, schema, binary, ...). I'm open to other suggestions though.

Reply:
generate-sdk? I'd rather be clear here it's not like this is a command where terseness is really needed.

Reply:
I'm good with `gen-sdk`. Looking over existing commands, we already have `gen-markdown` and `gen-completion` so there is precedent with just `gen-` (though, these are hidden).